platform/pulpcore/app/status.py

```diff
@@ -0,0 +1,65 @@
+"""
+Module to return status information about a running Pulp instance
+"""
+from pkg_resources import get_distribution
+import logging
+
+from pulpcore.app.models.task import Worker
+from pulpcore.tasking.celery_instance import celery
+
+_logger = logging.getLogger(__name__)
+
+
+def get_version():
+    """
+    Gets the current version of the pulpcore package
+
+    Returns:
+        str: the current version of pulpcore
+    """
+    return get_distribution("pulpcore").version
+
+
+def get_db_conn_status():
+    """
+    Returns True if pulp is connected to the database
+
+    Returns:
+        bool: True if there's a db connection. False otherwise.
+    """
+    try:
+        Worker.objects.count()
+        return True
```

Comment:
This could be moved to an `else` block because we're not trying to handle exceptions generated from this statement.